src/lib/mina.ts

```diff
@@ -55,12 +55,39 @@ interface TransactionId {
 }
 
 interface Transaction {
+  /**
+   * Transaction structure used to describe a state transition on the Mina blockchain.
+   */
   transaction: ZkappCommand;
+  /**
+   * Returns a JSON representation of the {@link Transaction}.
+   */
   toJSON(): string;
+  /**
+   * Returns a JSON representation of only the fields that differ from the original {@link Transaction}.
```

Comment:
this isn't really correct as the "original transaction" has no account updates.. maybe just "Returns a pretty-printed JSON representation of the Transaction"?